packages/core/src/App/Constants/routes-config.js

```diff
@@ -286,13 +293,34 @@ const getModules = () => {
                     component: Cashier,
                     getTitle: () => localize('Deriv P2P'),
                     icon_component: 'IcDp2p',
-                },
-                {
-                    path: routes.cashier_p2p_profile,
-                    component: Cashier,
-                    getTitle: () => localize('Deriv P2P'),
-                    icon_component: 'IcDp2p',
-                    is_invisible: true,
+                    routes: [
+                        {
+                            path: routes.p2p_buy_sell,
+                            component: P2P,
+                            getTitle: () => localize('Buy / Sell'),
+                            default: true,
+                        },
+                        {
+                            path: routes.p2p_advertiser_page,
+                            component: P2P,
+                            getTitle: () => localize("Advertiser's page"),
+                        },
+                        {
+                            path: routes.p2p_orders,
+                            component: P2P,
+                            getTitle: () => localize('Orders'),
+                        },
+                        {
+                            path: routes.p2p_my_ads,
+                            component: P2P,
+                            getTitle: () => localize('My ads'),
+                        },
+                        {
+                            path: routes.p2p_my_profile,
+                            component: P2P,
+                            getTitle: () => localize('My profile'),
+                        },
+                    ],
                 },
                 {
                     path: routes.cashier_p2p_verification,
```

Comment:
Shouldnt we rename this to `p2p_verification` ? Just to standardise the naming convention

Reply:
yeah agreed, we can also rename this 🙏